src/cljs/bluegenes/pages/results/events.cljs

```diff
@@ -163,3 +164,43 @@
                         :enrichment-results
                         (keyword (:widget params))] nil)
       :enrichment/get-enrichment [(:widget params) enrichment-chan]})))
+
+(reg-event-db
+ :description/edit
```

Comment:
can I suggest naming this edit-list? otherwise it's a tad vague.

Reply:
What do you think about `:list-description/edit`? This way we can prefix `list` to the other *description* events as well to keep their names consistent.

Reply:
perfect!